extensions/PhabBugz/lib/Util.pm

```diff
@@ -115,6 +117,10 @@ sub get_bug_role_phids {
   return [map { $_->phid } @{$phab_users}];
 }
 
+sub is_bug_assigned {
+  return $_->assigned_to->email ne 'nobody@mozilla.org';
```

Comment:
return $_[0]->assigned_to->email ne 'nobody@mozilla.org';

Once I added the [0], everything worked fine. r=dkl